make.py

```diff
@@ -108,40 +108,40 @@
 
 assert len(glob.glob('res/*.c')) == len(c_entry)
 
+def run(entry):
+    print(f'run bin/{entry}')
+    s = subprocess.call(f"ckb-debugger-rvv --bin bin/{entry} | fgrep 'Run result: 0'", shell=True)
```

Comment:
Don't use debugger here, the goal of rvv-playground is to execute test cases during development, using debugger will make it all very complicated.

Reply:
ok

Reply:
Can I use `rvv` branch of `examples/int64` ?

Reply:
yes, keep it as is